gateway/1.5/modules/ROOT/pages/policies-pdk-debug-deployed-policies.adoc

```diff
@@ -17,7 +17,7 @@ The PDK debugging environment provides an easy way to view Flex Gateway logging
 For more information about PDK's debugging environment, see xref:policies-pdk-debug-local.adoc[].
 
 [[debug-logs]]
-== Troubleshoot in a Production Environment
+== Debug in a Production Environment
 
 It is possible that errors that are not present in PDK's debugging environment are present when the policy is deployed to a production environment.
```

Comment:
```suggestion
Errors that are not present in the PDK debugging environment might be present when the policy is deployed to a production environment.
```

Reply:
Don't use possessives with product/feature names. 

Is PDK an approved initialism?